File: openapi/code_samples/nodejs/utilitypayments/pay_bill/code.js
var request = require('request');
var options = {
  'method': 'POST',
  'url': 'https://utilities.reloadly.com/pay',
  'headers': {
    'Accept': 'application/com.reloadly.utilities-v1+json',
    'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
    'Content-Type': 'application/json'
  },
  body: JSON.stringify({
    "subscriberAccountNumber": "04223568280",
    "amount": 1000,
    "billerId": 5,
    "useLocalAmount": false
  })

};
request(options, function (error, response) {
  if (error) throw new Error(error);
  console.log(response.body);
});

File: openapi/code_samples/python/airtime/get_operator_by_iso/code.py
import requests
import json

url = "https://topups.reloadly.com/operators/countries/PK"

payload={}
headers = {
  'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
  'Accept': 'application/com.reloadly.topups-v1+json',
  'Content-Type': 'application/json'
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)